src/components/SearchBarView.tsx

```diff
@@ -1,84 +1,91 @@
-import { gql, useLazyQuery } from '@apollo/client'
-import React, { useEffect, useState } from 'react'
-import { SafeAreaView, StyleSheet, View } from 'react-native'
-import { Icon, Input } from 'react-native-elements'
+import { gql, useLazyQuery } from "@apollo/client";
+import React, { useEffect, useState } from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import { Icon, Input } from "react-native-elements";
 
 // GraphQL fragment
-const COMPANY_TILE_DATA = gql` {
-    fragment CompanyTile on Business{
-        id
-        name
-        profilePicture
-        sustainabilityScore
-        customerScore
+const COMPANY_TILE_DATA = gql`
+  {
+    fragment
+    CompanyTile
+    on
+    Business {
+      id
+      name
+      profilePicture
+      sustainabilityScore
+      customerScore
     }
-}
-`
+  }
+`;
 
 const SearchBarView = () => {
-  const [searchText, onChangeText] = useState('')
-    
-  const [executeSearch, { error }] = useLazyQuery(    // 'loading' and 'data' can also be returned (not just error)
+  const [searchText, onChangeText] = useState("");
+
+  const [executeSearch, { error }] = useLazyQuery(
+    // 'loading' and 'data' can also be returned (not just error)
     COMPANY_TILE_DATA
-  )
+  );
 
   useEffect(() => {
     // Debounce query
     const delayDebounceFn = setTimeout(() => {
       executeSearch({
-        variables: { String: searchText }
-      }).then(data => console.log(data), () => console.error(error))
-    }, 300) 
-    return () => clearTimeout(delayDebounceFn)
-  }, [searchText])
+        variables: { String: searchText },
+      }).then(
+        (data) => console.log(data),
+        () => console.error(error)
+      );
+    }, 300);
+    return () => clearTimeout(delayDebounceFn);
+  }, [searchText]);
 
   return (
-    <SafeAreaView style={styles.safeAreaView}>
+    <React.Fragment>
```

Comment:
Why has this changed from a SafeAreaView?

Reply:
So that it can be embedded properly in the slide up panel.

Reply:
Yeah this doesn't need a safe area view anyway?